test/gemm-microkernel-tester.cc

```diff
@@ -1688,7 +1688,7 @@ void GemmMicrokernelTester::Test(
       for (size_t j = 0; j < n(); j++) {
         // Extract tolerance into variable to workaround test failures on Linux AArch64.
         const float tolerance =
-            std::max(2.5e-5f, std::abs(c_ref[i * n() + j]) * 2.5e-6f);
+            std::max(1.0e-4f, std::abs(c_ref[i * n() + j]) * 1.0e-5f);
```

Comment:
1.0e-5f is the typical tolerance for float gemm.  is it necessary to allow more error here?

Reply:
I agree, there's something fishy here.